### PR TITLE
Use new test interface definitions

### PR DESCRIPTION
**Pull request by jacobperron**
Connects to https://github.com/ros2/rcl_interfaces/issues/58

### CODE REVIEW

File: rclpy/test/test_node.py [hunk 1]
```diff
@@ -75,15 +75,15 @@ def raw_subscription_callback(self, msg):
     def test_create_raw_subscription(self):
         executor = SingleThreadedExecutor(context=self.context)
         executor.add_node(self.node)
-        primitives_pub = self.node.create_publisher(Primitives, 'raw_subscription_test')
+        primitives_pub = self.node.create_publisher(BasicTypes, 'raw_subscription_test')
```
Comment (dirk-thomas) [inline rclpy/test/test_node.py]: Maybe change the name of the variables too?

Same below: `primitives_msg`.

Comment (jacobperron) [reply rclpy/test/test_node.py]: Missed it! Changed be3848c

**Comment by sloretz** [comment]
@jacobperron FYI it looks like there's a small merge conflict (git doesn't know which order to add the new import statements)

**Comment by jacobperron** [comment]
@sloretz I've rebased and also updated the changes from master to use the new service name `BasicTypes`.

**Pull request merged**